sdk/formrecognizer/ai-form-recognizer/test/internal/transforms.spec.ts

```diff
@@ -2321,7 +2321,9 @@ describe("Transforms", () => {
       const original: FieldValueModel = {
         type: "phoneNumber",
         valuePhoneNumber: "phoneNumber value",
-        ...commonProperties
+        ...commonProperties,
+        // TODO: regression causes phone numbers to be sent in 'text'
+        text: "phoneNumber value"
```

Comment:
Are you sure this is a regression? I thought that you only got a value in the value field if the service was able to take the text and transform it into some specific phone number format like +1XXXXXXXXXX

Reply:
So should this be `valuePhoneNumber` if it's defined and `text` if it's not? is there any reason that `valuePhoneNumber`, if defined, should differ from `text`?

It just seems odd to me that the result would identify a field as a phone number (because the training schema explicitly specified it as such) and then a user would get an undefined `field.value`, but see the a correct-looking result in `field.valueData.text`. Almost feels like the response is contradicting itself by saying "I identified a phone number, but wasn't able to extract a phone number value."

Reply:
This is the response I got from the service team:

> We can have null for the value* field if we cannot normalize the text string into a standardized form.  In this specific case, we currently failed to normalize the phone number string "+44 (0) 7911 123456" into "+447911123456" because of the "(0)".  Thanks for reporting this quality bug.  We will improve the normalizer to be more robust in future releases.

I suppose that the .value should be in a specific format, and that users should expect that format when accessing it.

Reply:
Thanks! Makes sense. I'll revert this for this PR. I'll also add docs to `FormField` explaining this, but I'll take that in a separate PR.